tests/CoreFX/runtest/src/TestUtils/XUnit/ResultFormatter.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
```

Comment:
All source files must have a license header (like this one: https://github.com/dotnet/corert/blob/3781c96980e5959924f7d23c91682df43ddde1a6/samples/WebApi/Program.cs#L1)